static/app/components/events/interfaces/crashContent/exception/useSourceMapDebug.tsx

```diff
@@ -0,0 +1,108 @@
+import uniqBy from 'lodash/uniqBy';
+
+import type {ExceptionValue, Frame} from 'sentry/types';
+import {defined} from 'sentry/utils';
+import {QueryKey, useQuery, UseQueryOptions} from 'sentry/utils/queryClient';
+
+export interface SourceMapDebugResponse {
+  errors: SourceMapProcessingIssueResponse[];
+}
+
+interface SourceMapProcessingIssueResponse {
+  data: Record<string, string>;
```

Comment:
@scttcper It would be better if we could type data more strongly such that we know what fields to expect for each type of `SourceMapProcessingIssueType`. It's a more complicated type but it will lead to fewer errors I think.